suite-native/home-graph/jest.config.js

```diff
@@ -1,3 +1,5 @@
+const config = require('../../jest.config.native');
+
 module.exports = {
-    preset: '../../jest.config.base.js',
+    ...config,
```

Comment:
please use these configs in all native packages

Reply:
Shouldn't we add testing config when we need it? That was commented on last PR by @bouchja1. I think it makes sense to add it only when tests are present in the package.

Reply:
We should but if the jest.configs are already here we should at least update them to version that will actually works.